src/stats.jl

```diff
@@ -185,57 +200,43 @@ function describe(
     io::IO,
     chains::Chains;
     q = [0.025, 0.25, 0.5, 0.75, 0.975],
-    etype = :bm,
     kwargs...
 )
-    dfs = vcat(summarystats(chains; etype = etype, kwargs...),
-               quantile(chains; q = q, kwargs...))
-    return dfs
-end
-
-function _hpd(x::AbstractVector{<:Real}; alpha::Real=0.05)
-    n = length(x)
-    m = max(1, ceil(Int, alpha * n))
-
-    y = sort(x)
-    a = y[1:m]
-    b = y[(n - m + 1):n]
-    _, i = findmin(b - a)
-
-    return [a[i], b[i]]
+    stats = [summarystats(chains; kwargs...), quantile(chains; q = q, kwargs...)]
+    return stats
 end
 
 """
-    hpd(chn::Chains; alpha::Real=0.05, kwargs...)
+    hdi(chn::Chains; prob::Real=0.94, kwargs...)
 
-Return the highest posterior density interval representing `1-alpha` probability mass.
+Return the unimodal highest density interval (HDI) representing `prob` probability mass.
```

Comment:
What's the meaning of "unimodal" here?

Reply:
The default estimator used by `hdi` assumes that the distribution is unimodal. The alternative (not yet in PosteriorStats but doable with HighestDensityRegions.jl) for multimodal distributions first fits a KDE and then partitions into one or more intervals by density. For fast summary statistics, the unimodal version is more useful.